.circleci/handle_request.sh

```diff
@@ -60,8 +76,8 @@ INDICATOR_FILENAME="submitted_data/request.handled"
 rm -rf $INDICATOR_FILENAME
 
 # For each yaml file, download and process data
-for f in ${yml_files[*]}; do
-    cimr processor -process -yaml-file $f
+for f in ${SUBMITTED_YAMLS}; do
+    cimr processor -process -yaml-file $f -catalog PR_catalog.txt
```

Comment:
-catalog-name PR_catalog.txt

Reply:
How come I used `-catalog foo.txt` and `cimr` doesn't complain?

Reply:
you are right, it doesn't! I did change -catalog to -catalog-name at some point but I must have missed an alias somewhere. I will just do some checks and make sure but this is fine.

Reply:
I changed it to `-catalog-name` to make the option more consistent.

Reply:
sounds good. thank you!